### PR TITLE
Feature/WAYDES-17 Documentation for way-select and all sub components

### PR DESCRIPTION
**Pull request by elwinvaneede**

### CODE REVIEW

**Comment by elwinvaneede** [comment]
The pre-rendered components don't seem to work correctly (forgot to test with `npm run serve`), I will look into this.

**Comment by elwinvaneede** [comment]
I've disabled the Stencil SSG/hydration of the Web Components for now, because client side hydration doesn't work well for all components (double elements in the shadow dom, resulting in double dividers for example). I haven't been able to solve this after quite some effort. Besides we don't really need it for this documentation site (although it would be nice, but for now I rather put my time in finishing the rest of the components).

**Comment by elwinvaneede** [comment]
> I've disabled the Stencil SSG/hydration of the Web Components for now, because client side hydration doesn't work well for all components (double elements in the shadow dom, resulting in double dividers for example). I haven't been able to solve this after quite some effort. Besides we don't really need it for this documentation site (although it would be nice, but for now I rather put my time in finishing the rest of the components).

This introduces a subtle layout shift because of the lazy loading, I've addressed this in a separate PR by bundling the custom-elements dist instead of the default lazy loading dist: #8

File: src/_includes/layouts/base.ejs [hunk 1]
```diff
@@ -71,20 +73,14 @@
...
         <div class="flex flex-col w-64">
           <div class="flex flex-col flex-grow pt-5 pb-4 overflow-y-auto border-r border-gray-200">
-            <div class="flex items-center flex-shrink-0 px-4">
-              <%- include('../logo.ejs') %>
-            </div>
-            <div class="flex flex-col flex-grow mt-5">
-              <%- include('../menu.ejs') %>
-            </div>
+            <div class="flex items-center flex-shrink-0 px-4"><%- include('../logo.ejs') %></div>
+            <div class="flex flex-col flex-grow mt-5"><%- include('../menu.ejs') %></div>
```
Comment (Tom-w2w) [inline src/_includes/layouts/base.ejs:77]: Zijn dit auto lint fixes? of jij zelf gedaan?  Ik vind het zo minder lekker lezen namelijk.

**Review by Tom-w2w — APPROVED** [review]

File: src/_includes/layouts/base.ejs (hunk 1 above)
Comment (elwinvaneede) [reply src/_includes/layouts/base.ejs:77]: > Zijn dit auto lint fixes? of jij zelf gedaan? Ik vind het zo minder lekker lezen namelijk.

Ja Prettier, wat dit project gebruikt. Dan moet je de prettierrc anders instellen, bijv. een printWidth van 80.

Comment (elwinvaneede) [reply src/_includes/layouts/base.ejs:75]: Maar ja eens, dat eens dat het wat minder lekker lezen is, maar vindt dat vooral komen door de Tailwind classes persoonlijk :).

**Pull request merged**